static/css/graphspace.css

```diff
@@ -128,16 +128,37 @@ div.container-fluid {
     position: absolute;
 }
 
-#cyGraphContainer {
+#graph{
```

Comment:
Lets not call it `graph`. It is very generic. ID should be descriptive. Call it `graphContainer`